src/components/FeedbackComponent/FeedbackComponent.tsx

```diff
@@ -1,14 +1,16 @@
 import { faCommentAltLines } from '@fortawesome/pro-regular-svg-icons'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+// eslint-disable-next-line
```

Comment:
@Stefwint highly recommend this to abstract this to a utility/helper to prevent multiple pages doing an import like this, you can also give some context in the helper as to why the import require that disable line